### DISCUSSION TITLE
Filter by Created OR Assigned

### DISCUSSION THREAD

**Discussion by Cossey** [community/community#5745] [Projects and Issues]
Unfortunately the advanced search does not support `OR`. 

Often I would like to see an Issue list with open issues created by me or assigned to me. Normally, I have to toggle between both Created and Assigned views to see any Issues that I need to follow up or keep an eye on. I kind of use the view as a "To Do" list and having both in a list would be helpful.

Following the GitHub filter syntax I want to filter something like this `is:open is:issue archived:false (assignee:Cossey OR author:Cossey)`.

**Comment by WizardCM** [comment]
I would also make use of an OR option. While prioritising PRs, I have a project table with "Complexity" and "Priority" columns, and would like to be able to filter `no:complexity OR no:priority` so I know which ones need to be evaluated.
